extensions/flyway/runtime/src/main/java/io/quarkus/flyway/runtime/FlywayDataSourceRuntimeConfig.java

```diff
@@ -149,7 +149,7 @@ public static final FlywayDataSourceRuntimeConfig defaultConfig() {
      * the validation step will fail.
      */
     @ConfigItem
-    public boolean ignoreFutureMigrations;
+    public boolean ignoreFutureMigrations = true;
```

Comment:
I think you also need to change the defaultValue of the ConfigItem annotation if you want to do that.

Reply:
Ah good catch